src/script/components/list/ConversationListCallingCell.tsx

```diff
@@ -0,0 +1,378 @@
+                    </div>
+                  )}
+                </div>
+
+                <div className="conversation-list-calling-cell-controls-right">
+                  {showParticipantsButton && (
+                    <div
+                      className={cx('call-ui__button call-ui__button--participants', {
+                        'call-ui__button--active': showParticipants,
+                      })}
+                      onClick={() => setShowParticipants(current => !showParticipants)}
+                      data-uie-name="do-toggle-participants"
+                    >
+                      <span>{t('callParticipants', participants.length)}</span>
+                      <Icon.Chevron className="chevron" />
+                    </div>
+                  )}
+                  {(isIncoming || isOutgoing) && (
+                    <div
+                      className="call-ui__button call-ui__button--red call-ui__button--large"
+                      onClick={() => (isIncoming ? callActions.reject(call) : callActions.leave(call))}
+                      title={t('videoCallOverlayHangUp')}
+                      data-uie-name="do-call-controls-call-decline"
+                    >
+                      <Icon.Hangup className="small-icon" style={{maxWidth: 17}} />
+                    </div>
+                  )}
+                  {isIncoming && !isDeclined && (
+                    <div
+                      className="call-ui__button call-ui__button--green call-ui__button--large"
+                      onClick={() => callActions.answer(call)}
+                      data-uie-name="do-call-controls-call-accept"
+                    >
+                      <Icon.Pickup className="small-icon" />
+                    </div>
+                  )}
+                </div>
+              </div>
+              <div
+                className={cx('call-ui__participant-list__wrapper', {
+                  'call-ui__participant-list__wrapper--active': showParticipants,
+                })}
+              >
+                <div
+                  className="call-ui__participant-list"
+                  ref={setScrollbarRef}
+                  data-uie-name="list-call-ui-participants"
+                >
+                  {(participants as Participant[])
+                    .slice()
+                    .sort((participantA, participantB) => sortUsersByPriority(participantA.user, participantB.user))
+                    .map(participant => (
+                      <ParticipantItem
+                        key={participant.clientId}
+                        participant={participant.user}
+                        hideInfo
+                        noUnderline
+                        callParticipant={participant}
+                        selfInTeam={selfUser?.inTeam()}
+                        isSelfVerified={isSelfVerified}
+                        external={teamRepository.isExternal(participant.user.id)}
```

Comment:
I think `isExternal` could be extracted to the `TeamState`. Thus we could use the `TeamState` and prevent using the whole `TeamRepository`.